src/anbox/cmds/session_manager.cpp

```diff
@@ -131,6 +131,11 @@ anbox::cmds::SessionManager::SessionManager()
                       use_system_dbus_));
 
   action([this](const cli::Command::Context &) {
+
+    Log().SetLogFile(std::string("/var/log/anbox/session.txt"));
+    Log().SetSeverityFromString("debug");
```

Comment:
Nope, this is wrong. Log level is influenced by specifying ANBOX_LOG_LEVEL= on the command line. We can in addition add a command line argument `--log-level=<level>`.